app/main/forms.py

```diff
@@ -1392,35 +1392,43 @@ def __init__(self, *args, **kwargs):
     )
 
 
-class BrandingOptionsEmail(StripWhitespaceForm):
+class BrandingOptions(StripWhitespaceForm):
 
     FALLBACK_OPTION_VALUE = 'something_else'
     FALLBACK_OPTION = (FALLBACK_OPTION_VALUE, 'Something else')
 
-    options = RadioField('Choose your new email branding')
+    options = RadioField('Choose your new branding')
     something_else = TextAreaField('Describe the branding you want')
 
-    def __init__(self, service, *args, **kwargs):
+    def __init__(self, service, *args, branding_type="email", **kwargs):
         super().__init__(*args, **kwargs)
-        self.options.choices = tuple(self.get_available_choices(service))
+        self.options.choices = tuple(self.get_available_choices(service, branding_type))
```

Comment:
You could keep the label text for the email choices the same by setting it in the constructor:
```suggestion
        self.options.choices = tuple(self.get_available_choices(service, branding_type))
        self.options.label.text = 'Choose your new {} branding'.format(branding_type)
```